binance_trade_bot/database_warmup.py

```diff
@@ -58,7 +59,7 @@ def set_coins_to_warmup(self, symbols: List[str], warmup_symbols: List[str]):
             #select all pairs with non exiting pair entry in db and add the missing pair entries
             query = session.query(c1, c2).\
                 join(c2, c2.symbol != c1.symbol).\
-                outerjoin(p, p.from_coin_id == c1.symbol and p.to_coin_id == c2.symbol).\
+                outerjoin(p, and_(p.from_coin_id == c1.symbol, p.to_coin_id == c2.symbol)).\
```

Comment:
@shanejones You where right! 👍   
The sql query was not generated properly because I did not use the and_ expression of sqlalchemy... This resulted in missing pairs and so no ratios...

Thanks for the hint 😄